packages/salesforcedx-vscode-core/test/vscode-integration/commands/forceProjectCreate.test.ts

```diff
@@ -123,6 +125,16 @@ describe('Force Project Create', () => {
         expect.fail('Response should be of type ContinueResponse');
       }
     });
+
+    it('Should return Continue with trimmed project name if project name input has leading and or trailing spaces', async () => {
+      const gatherer = new SelectProjectName();
+      const response = await gatherer.gather();
+      if (response.type === 'CONTINUE') {
```

Comment:
nit: I'd let expect do it's job and go with 
```
expect(response.type).to.equal('CONTINUE');
expect(response.data.projectName).to.equal(PROJECT_NAME);
```
 and do away with the condition.